### PR TITLE
module: avoid error when require an empty module fix/gh-330.

### PR DESCRIPTION
**Pull request by Txiaozhe**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by Txiaozhe** [comment]
#330

File: src/modules/iotjs_module_process.c [hunk 1]
```diff
@@ -146,10 +146,6 @@ JS_FUNCTION(Compile) {
 
   size_t size = 0;
   char* source = iotjs__file_read(filename, &size);
-  if (source == NULL || size == 0) {
```
Comment (yorkie) [inline src/modules/iotjs_module_process.c]: I think NULL is also needed to be checked :)

**Comment by yorkie** [comment]
And BTW, if you are going to fix an issue like #330, then the branch name should be fix/gh-330 :)

File: src/modules/iotjs_module_process.c (hunk 1 above)
Comment (yorkie) [reply src/modules/iotjs_module_process.c]: See https://github.com/Rokid/ShadowNode/blob/master/src/iotjs_util.c#L30, the NULL means the file doesn't exist.

File: src/modules/iotjs_module_process.c [hunk 2]
```diff
@@ -146,12 +146,11 @@ JS_FUNCTION(Compile) {
 
   size_t size = 0;
   char* source = iotjs__file_read(filename, &size);
+  jerry_value_t jres = WrapEval(filename, strlen(filename), source, size);
```
Comment (yorkie) [inline src/modules/iotjs_module_process.c]: We should verify the source before `WrapEval`, what I meant is to throw the error when source is NULL.

**Pull request merged**